### DISCUSSION TITLE
Add development progress bar in this site or game.

### DISCUSSION THREAD

**Discussion by ghost** [redeclipse/base#1282] [Ideas and Feature Requests]
Like 0 A.D, a free and open sourse RTS game. Every version they plan to develop have a progress bar, and they also have such as "fuction freeze", "debug freeze", "translation freeze". 
![Screenshot 2021-07-09 at 10-13-10 Roadmap – Wildfire Games](https://user-images.githubusercontent.com/87136890/125013649-c76a0200-e09e-11eb-822d-9d62738087f2.png)

**Comment by cloudziq** [comment]
there is much difference between devs there and here....
